src/module.c

```diff
@@ -117,14 +118,13 @@ NAPI_METHOD(dcn_set_event_handler) {
   napi_value async_resource_name;
   NAPI_STATUS_THROWS(napi_create_string_utf8(env, "dc_event_callback", NAPI_AUTO_LENGTH, &async_resource_name));
 
-  //TODO: Figure out how to release threadsafe_function
   NAPI_STATUS_THROWS(napi_create_threadsafe_function(
     env,
     callback,
     0,
     async_resource_name,
-    0,
-    3,
+    100,
```

Comment:
We need to set _something_ for queue size. If we set to 0 it means we get an unlimited queue and this will crash.